### DISCUSSION TITLE
【委托】求助角色抽取建议

### DISCUSSION THREAD

**Discussion by SKWTEA** [share121/inter-knot#3314] [其他]
本人绳网小萌新一枚，现在有莱卡恩、苍角、艾莲，加密母带120抽，在考虑抽简•杜或者卡吕东之子的那个火异常（忘了叫啥了），望绳网的大佬们给点建议

**Comment by Guessme404** [comment]
建议去米游社问，这里回帖不容易看到，而且你除了冰队另一对呢？

**Reply by SKWTEA** [reply]
除了冰队现在还没有二队🤣
